lib/simulator.js

```diff
@@ -23,23 +23,36 @@ function handleUnsupportedXcode (xcodeVersion) {
  * Finds and returns the corresponding Simulator instance for the given ID.
  *
  * @param {string} udid - The ID of an existing Simulator.
+ * @param {Object} opts - an Options object that can include the `platform` of
+ *                        the simulator associated with the udid, and `checkExistance`,
+ *                        which allows one to skip checking that the udid is valid
+ *                        (defaults to `true`)
  * @throws {Error} If the Simulator with given udid does not exist in devices list.
  *   If you want to create a new simulator, you can use the `createDevice()` method of
  *   [node-simctl](github.com/appium/node-simctl).
  * @return {object} Simulator object associated with the udid passed in.
  */
-async function getSimulator (udid) {
+async function getSimulator (udid, opts = {}) {
+  let {
+    platform,
```

Comment:
should platform be equal to `iOS` by default?

Reply:
Sure.